msal-objc-articles/reference/Classes/class_m_s_a_l_silent_token_parameters.md

```diff
@@ -0,0 +1,14 @@
+---
+title: MSALSilentTokenParameters
+
+---
+
+# MSALSilentTokenParameters
+
+
```

Comment:
This problem exists across a few classes - the public `@property` (properties) are missing

Example:
- `forceRefresh` not documented here
- `allowUsingLocalCachedRtWhenSsoExtFailed` not documented here

Reference header is here:
https://github.com/AzureAD/microsoft-authentication-library-for-objc/blob/dev/MSAL/src/public/MSALSilentTokenParameters.h